src/main/java/org/fcrepo/sword/http/SWORDServiceResource.java

```diff
@@ -28,15 +31,25 @@
  */
 @Scope("request")
```

Comment:
Do you need the `sword` service to create a new instance of this class with every request? or can the class be a singleton within the container? Setting the `@Scope` to `prototype` allows your current tests to pass successfully.
If you need `Scope("request")` we will need to dig into why that is not currently working.